res/schema.xml

```diff
@@ -480,5 +480,30 @@ METADATA
         position INTEGER
         );
     </sql>
-  </revision>  
+  </revision>
+  <revision version="31" min_compatible="3">
+    <description>
+      Convert the PredefinedColor id to the actual RGBA value.
+    </description>
+    <sql>
+      <!-- No Color becomes black 0x000000 -->
+      UPDATE cues SET color=0 WHERE color=0;
```

Comment:
SQLite understands hex numbers and we should use this notation here:
https://www.sqlite.org/syntax/numeric-literal.html

Reply:
Cool, I was not aware of this.